packages/main/src/resolvers/conversation.ts

```diff
@@ -91,6 +97,30 @@ export const ConversationMutations: ConversationMutationsResolvers = {
     return thread
   },
 
+  async flag(_parent, { ids, isFlagged }) {
+    let threads: C.Conversation[] = []
+    ids.forEach(id => {
+      const thread = C.mustGetConversation(id)
+      updateAction(thread.messages, (accountId, box, uids) => {
+        schedule(
+          isFlagged
```

Comment:
I understand the thinking here; but I think that `isFlagged` should reflect the result that you want. So if `isFlagged` is `true` then this resolver should add a star.